maths/euclidean_distance.py

```diff
@@ -0,0 +1,27 @@
+from typing import List, Tuple, Union
+
+import numpy as np
+
+Vector = Union[List[float], List[int], Tuple[float], Tuple[int], np.ndarray]
```

Comment:
```suggestion
from typing import Iterable Union

import numpy as np

Vector = Union[Iterable[float], Iterable[int], np.ndarray]
```
Would mypy consider a np.ndarray considered an Iterable